File: prototype/rails/src/app/domain/escrow/fee.rb
require_relative "../money"

module Domain
  module Escrow
    module Fee
      PLATFORM_PERCENT = 10

      module_function

      def platform(subtotal)
        subtotal.percent(PLATFORM_PERCENT)
      end

      def net(subtotal)
        Money.from_cents(subtotal.cents - platform(subtotal).cents)
      end
    end
  end
end

File: prototype/php/src/app/Domain/Escrow/Fee.php
<?php

declare(strict_types=1);

namespace App\Domain\Escrow;

use App\Domain\Money\Money;

final class Fee
{
    public const PLATFORM_PERCENT = 10;

    public static function platform(Money $subtotal): Money
    {
        return $subtotal->percent(self::PLATFORM_PERCENT);
    }

    public static function net(Money $subtotal): Money
    {
        return $subtotal->subtract(self::platform($subtotal));
    }
}
